### ISSUE TITLE
shouldn't this be noarch?

### ISSUE THREAD

**Issue by ericdill** [conda-forge/depfinder-feedstock#35]
### Comment:

@conda-forge-admin, please add noarch: python

**Comment by ericdill** [comment]
i guess i always forget that noarch: python means you can't have any selectors at all anywhere. blergh

**Comment by ocefpaf** [comment]
> i guess i always forget that noarch: python means you can't have any selectors at all anywhere. blergh

That is not True with the new "semi-arch" but not for Python as far as I know. TL;DR we should issue a new version of depfinder that is Python >=3.10 so we can make it noarch again.
